Resources/public/js/views/ez-trashview.js

```diff
@@ -96,6 +99,44 @@ YUI.add('ez-trashview', function (Y) {
         },
 
         /**
+         * Finds a trash item in the item list and adds it to a result array
+         *
+         * @private
+         * @method _findTrashItems
+         * @param {String} trashItemId
+         * @param {Array} resultArray Item will be added to this array if it is found
+         */
+        _findTrashItems: function (trashItemId, resultArray) {
```

Comment:
it's weird to have the result in a parameter of such method, why not returning the result ?